rrule.gemspec

```diff
@@ -21,6 +21,7 @@ Gem::Specification.new do |s|
     'changelog_uri' => 'https://github.com/square/ruby-rrule/blob/master/CHANGELOG.md'
   }
 
-  s.required_ruby_version = '>= 2.3.0'
-  s.add_runtime_dependency 'activesupport', '>= 4.1'
+  s.required_ruby_version = '>= 2.4.0'
+  s.add_runtime_dependency 'activesupport'
```

Comment:
I think it would be good practice to use

```ruby
s.add_runtime_dependency 'activesupport', '>= 2.3.0', '< 7'
```

nobody can foresee whether the gem will work correctly with future versions of ActiveSupport.

Reply:
This makes sense to me.